### DISCUSSION TITLE
Server-Side GTM?

### DISCUSSION THREAD

**Discussion by eyesurewould** [gtm-support/vue-gtm#74] [Q&A]
Does this package support the new [Server-Side Tagging](https://developers.google.com/tag-platform/tag-manager/server-side) feature of GTM?

**Comment by Shinigami92** [comment]
What would be needed to support it?

**Reply by matteplus** [reply]
Being able to specify a server URL. I am also interested in this enhancement.
I guess that for the time being we can just set 
`loadScript: false`
and import gtm.js manually.

**Comment by tomreichenberger** [comment]
I'm also very interrest in that - any updates on this side?
